js/assessment.js

```diff
@@ -247,6 +247,32 @@ define([
             }
         },
 
+        _setupQuestionNumbering: function() {
+            var getRelatedQuestions = function(model) {
+                var currentAssessmentId = model._assessmentId;
+                var currentAssessment =  Adapt.assessment.get(currentAssessmentId);
+                return currentAssessment.getState().questionModels;
+            };
+
+            Handlebars.registerHelper("questionNumber", function getQuestionNumber() {
+                var model = this.view ? this.view.model.attributes : this;
+                if (!model._isPartOfAssessment) return;
+                
+                var related = getRelatedQuestions(model)
+                    .map(function(question) {
+                        return question.attributes._id;
```

Comment:
`getRelatedQuestions()` returns a collection so you can use just `.pluck('_id')`.